extra-multimedia/feeluown-netease/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME="feeluown-netease"
+PKGSEC=sound
+PKGDES="a FeelUOwn plugin for the Netease Cloud Music support"
```

Comment:
First letter in `PKGDES=` should be capitalised; the "A" is actually not necessary here.